main/fossil/APKBUILD

```diff
@@ -1,16 +1,16 @@
 # Contributor: Paul Onyschuk <blink@bojary.koba.pl>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=fossil
-pkgver=2.5
-pkgrel=1
+pkgver=2.6
+pkgrel=0
 pkgdesc="Simple, high-reliability, distributed software configuration management"
 url="http://www.fossil-scm.org"
 arch="all"
 license="BSD"
+options="!check" # missing dependencies; broken tests
 depends=""
 makedepends="libressl-dev sqlite-dev tcl-dev zlib-dev"
 source="http://www.fossil-scm.org/index.html/uv/$pkgname-src-$pkgver.tar.gz"
-builddir="$srcdir"/$pkgname-$pkgver
```

Comment:
Since the majority of aports explicitly set the `builddir` variable, it would be good form to just keep it.